### PR TITLE
add observer to allow custom handling fees

### PR DESCRIPTION
**Pull request by zenexpert**
Allow custom handling fees based on whatever criteria you may need (ie. certain products only)

### CODE REVIEW

File: includes/modules/shipping/usps.php [hunk 1]
```diff
@@ -718,6 +718,11 @@ public function quote($method = '')
                 // international
                 $usps_handling_fee = MODULE_SHIPPING_USPS_HANDLING_INT;
             }
+            
+            // -----
+            // Give an observer the opportunity to modify handling costs.
+            //
+            $this->notify('NOTIFY_SHIPPING_USPS_AFTER_HANDLING', array(), $order, $usps_shipping_weight, $shipping_num_boxes, $usps_handling_fee);
```
Comment (lat9) [inline includes/modules/shipping/usps.php:725]: Please update to use 'short array syntax'.

Comment (zenexpert) [reply includes/modules/shipping/usps.php:725]: Now you got me confused... 
Isn't long array syntax the norm in ZC? Haven't seen a single short array syntax in core code (with notifiers), and this one is in line with all other notifiers in the file...

Comment (lat9) [reply includes/modules/shipping/usps.php:725]: We're migrating towards short-array syntax.

**Comment by lat9** [comment]
I'm closing the PR and will include the notification (with short-array syntax) for the K11f release.

**Pull request closed without merging**